include/SFML/Window/Event.hpp

```diff
@@ -154,6 +154,39 @@ class SFML_WINDOW_API Event
         Vector2i position; //!< Position of the mouse pointer, relative to the top left of the owner window
     };
 
+    ////////////////////////////////////////////////////////////
+    /// \brief Mouse move raw event
+    ///
+    /// Raw mouse input data comes unprocessed from the
+    /// operating system hence "raw". While the MouseMoved
+    /// position value is dependent on the screen resolution,
+    /// raw data is not. If the physical mouse is moved too
+    /// little to cause the screen cursor to move at least a
+    /// single pixel, no MouseMoved event will be generated. In
+    /// contrast, any movement information generated by the
+    /// mouse independent of its sensor resolution will always
+    /// generate a MouseMovedRaw event.
+    ///
+    /// In addiiton to screen resolution independence, raw
```

Comment:
```suggestion
    /// In addition to screen resolution independence, raw
```

Small typo, but the new description is really good and informative!!

Reply:
Fixed.